tests/pytests/test_query_while_flush.py

```diff
@@ -0,0 +1,133 @@
+import threading
+import time
+from common import *
+
+@skip(cluster=True)
+def test_query_while_flush(env):
```

Comment:
We should create an `env` with workers, no?
Also consider using `getWorkersThpoolStats` for validation

Reply:
oh sure, will create with WORKERS